### PR TITLE
druid.indexer.task.restoreTasksOnRestart configuration.

### PR DESCRIPTION
**Pull request by gianm**

### CODE REVIEW

**Comment by nishantmonu51** [comment]
:+1:

File: indexing-service/src/main/java/io/druid/indexing/common/config/TaskConfig.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
```
Comment (drcrallen) [inline indexing-service/src/main/java/io/druid/indexing/common/config/TaskConfig.java]: unused import?

**Comment by drcrallen** [comment]
:+1: after minor fix and travis

**Comment by drcrallen** [comment]
This came out of a conversation about https://github.com/druid-io/druid/pull/2141

File: indexing-service/src/main/java/io/druid/indexing/common/config/TaskConfig.java (hunk 1 above)
Comment (gianm) [inline indexing-service/src/main/java/io/druid/indexing/common/config/TaskConfig.java]: fixed

**Pull request merged**